CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.11)
+project(conway_gol)
+
+set(CMAKE_CXX_STANDARD 20)
+
+find_package(SFML COMPONENTS graphics REQUIRED)
+
+add_executable(${PROJECT_NAME})
+target_link_libraries(${PROJECT_NAME} PRIVATE sfml-graphics)
+
+target_include_directories(${PROJECT_NAME} PRIVATE
+src
+)
+target_sources(${PROJECT_NAME} PRIVATE
+
+src/main.cpp
+src/Grid.hpp
```

Comment:
It seems the entire CMake script lacks indentation, this will get harder to read as it evolves over time. Recommend incorporating indents and formatting.

Reply:
added!